Jenkinsfile

```diff
@@ -0,0 +1,74 @@
+#! groovy
+@Library('corda-shared-build-pipeline-steps@5.0.1') _
+
+import com.r3.build.enums.BuildEnvironment
+import com.r3.build.utils.GitUtils
+
+int cpus = 1
+BuildEnvironment buildEnvironment = BuildEnvironment.AMD64_LINUX
+
+GitUtils gitUtils = new GitUtils(this)
+
+pipeline {
+    agent {
+        kubernetes {
+            cloud "eks-e2e"
+            yaml kubernetesBuildAgentYaml('build', buildEnvironment, cpus)
+            idleMinutes 15
+            podRetention always()
+            nodeSelector ([
+                    "kubernetes.io/arch=${buildEnvironment.buildArchitecture.toString().toLowerCase()}",
+                    "kubernetes.io/os=${buildEnvironment.buildOperatingSystem.toString().toLowerCase()}",
+                    "role=jenkins-agent"
+            ].join(','))
+            label ([
+                    "kubernetes-build-agent",
+                    "${cpus}cpus",
+                    "${buildEnvironment.buildArchitecture.toString().toLowerCase()}",
+                    "${buildEnvironment.buildOperatingSystem.toString().toLowerCase()}"
+            ].join('-'))
+            showRawYaml false
+            defaultContainer 'build'
+        }
+    }
+
+  parameters {
+    booleanParam defaultValue: (gitUtils.isReleaseBranch() || gitUtils.isReleaseTag(env.TAG_NAME)), description: 'Publish artifacts to Artifactory?', name: 'DO_PUBLISH'
+  }
+
+    environment {
+        GRADLE_PERFORMANCE_TUNING = '--parallel --build-cache'
+        GRADLE_USER_HOME = "/host_tmp/gradle"
+        ARTIFACTORY_CREDENTIALS = credentials('artifactory-credentials')
+        CORDA_ARTIFACTORY_PASSWORD = "${env.ARTIFACTORY_CREDENTIALS_PSW}"
+        CORDA_ARTIFACTORY_USERNAME = "${env.ARTIFACTORY_CREDENTIALS_USR}"
+        CORDA_ARTIFACTORY_REPOKEY = "${isRelease ? 'corda-dependencies' : 'corda-dependencies-dev'}"

```

Comment:
CORDA_ARTIFACTORY_REPOKEY is not being used in this project. 

See

https://github.com/corda/gradle-capsule-plugin/blob/arshadm/ent-6914-gradle-7-upgrade/build.gradle#L89 

2 things here 

- update the line i linked to pick up this env var as the lines below it do here is [another example](https://github.com/corda/CNO-Extension/blob/release/0.2/build.gradle#L202) from elsewhere, note the last property passed to  System.getProperty is a default value that is used if nothing else can be found, in our case use the dev repo  `corda-dependencies-dev`
- isRelease now no longer references a valid method, should be the gitutils method